lib/less-parser.js

```diff
@@ -116,7 +116,7 @@ export default class LessParser extends Parser {
 
       if (node.selector.indexOf('!important') >= 0) {
         node.important = true;
-        node.selector = node.selector.replace(/\s!important/, '');
+        node.selector = node.selector.replace(/\s*!important/, '');
```

Comment:
shouldn't this regex be `/(\s+)?!important/` ?

Reply:
That's equivalent, no? We can also use `/\s?!important/` but I though it would make sense to eat as much whitespace as possible here